firstvoices/backend/search/indexing/media_index.py

```diff
@@ -13,6 +13,13 @@ class MediaDocumentManager(DocumentManager):
     @classmethod
     def create_index_document(cls, instance):
         """Returns a MediaDocument populated for the given media instance."""
+
+        # Handle the case where the media instance has no original file
+        if instance.original:
```

Comment:
Do we want to index entries without a file present ? As if we index entries without the media, during search, the results may appear broken to the users.

Reply:
Currently the media models allow the `original` field to be null, I believe for migration reasons. A ticket has been made to research whether we should change the model (https://firstvoices.atlassian.net/browse/FW-5676) but this PR assumes that the `original` can still be allowed to be null

Reply:
This is a good question... I'm pretty sure the media dashboard is using the search API, so if we ever want to be able to remove these blank models we need to keep indexing them for now